compiler/circle-execution-plan/pal/IScratchpadHelper.h

```diff
@@ -26,8 +29,16 @@ namespace circle_planner
 class IScratchpadHelper
 {
 public:
+  virtual uint32_t
+  ComputeScratchpadSizeAveragePool2d(const luci::CircleAveragePool2D *avg_pool) = 0;
```

Comment:
Just noticed that this naming is different from other places, where you use "snake_case" style.